parfive/downloader.py

```diff
@@ -58,11 +61,10 @@ class Downloader:
     """
 
     def __init__(self, max_conn=5, progress=True, file_progress=True,
-                 loop=None, notebook=None, overwrite=False):
+                 loop=None, notebook=None, overwrite=False, ua_header=None, proxy=None, proxy_auth=None):
```

Comment:
On the headers front, we should have a kwarg for generic headers (which is the same format as the `headers=` keyword argument to `aiohttp.ClientSession.get` (see [here](https://docs.aiohttp.org/en/stable/client_advanced.html#custom-request-headers)).